keras_cv/bounding_box/utils.py

```diff
@@ -87,3 +87,110 @@ def _format_outputs(boxes, squeeze):
     if squeeze:
         return tf.squeeze(boxes, axis=0)
     return boxes
+
+
+def pad_with_sentinels(bounding_boxes):
+    """Pads the given bounding box tensor with -1s.
+
+    This is done to convert RaggedTensors to be converted into
+    standard Dense tensors, which have better performance and
+    compatibility within the TensorFlow ecosystem.
+
+    Args:
+        bounding_boxes: a ragged tensor of bounding boxes.
+        Can be batched or unbatched.
+
+    Returns:
+        a Tensor containing the -1 padded bounding boxes.
+    """
+    return bounding_boxes.to_tensor(-1)
+
+
+def filter_sentinels(bounding_boxes):
+    """converts a Dense padded bounding box `Tensor` to a `tf.RaggedTensor`.
+
+    Args:
+        bounding_boxes: a Tensor of bounding boxes.  May be batched, or unbatched.
+
+    Returns:
+        `tf.RaggedTensor` containing the filtered bounding boxes.
+    """
+    is_ragged = isinstance(bounding_boxes, tf.RaggedTensor)
+    is_batched = len(bounding_boxes.shape) == 3
+    if is_batched:
+        if is_ragged:
+            bounding_boxes = bounding_boxes.to_tensor(-1)
```

Comment:
call the utility above for clarity:
```
    bounding_boxes = pad_with_sentinels(bounding_boxes)
```

Good to spread good patterns within our own code base!

Reply:
yess! updated.